templates/analytics/stats.html

```diff
@@ -71,6 +71,14 @@ <h1 class="analytics-page-header">Zulip Analytics for {{ realm_name }}</h1>
       </div>
     </div>
   </div>
+  <div class="center-container" id="full_update_anchor">
+    <div class="center-block">
+      <p class="graph-title">
+        Last Update : <span class="last_full_update"></span>
+        <span class="icon-vector-question-sign" title="{{ _('Full updates of all the graphs happen once a day. The “Messages Sent Over Time” graph is updated once an hour.') }}"></span>
```

Comment:
-> "A full update of all the graphs happens" (I know I gave this incorrectly before)